activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -140,6 +140,10 @@ def humanize(lower_case_and_underscored_word, options = {})
       result
     end
 
+    def capitalize_first string
```

Comment:
convention is to use parentheses in method names, `def capitalize_first(string)`